src/transformers/models/beit/configuration_beit.py

```diff
@@ -139,14 +150,16 @@ def __init__(
         layer_scale_init_value=0.1,
         drop_path_rate=0.1,
         use_mean_pooling=True,
-        out_indices=[3, 5, 7, 11],
+        semantic_out_indices=[3, 5, 7, 11],
```

Comment:
We can't make this change because of backwards compatibility. If someone loads in their config and they don't have the default value for `out_indices` then their model's behaviour will have changed. Moreover, if they try to set or change `out_indices`, which they might have done in their own code, this won't be correctly updated here.